tutorial/markdown/java/spring-data.md

```diff
@@ -19,37 +19,71 @@ sdk_language:
 length: 30 Mins
 ---
 
-[![Try it now!](https://da-demo-images.s3.amazonaws.com/runItNow_outline.png?couchbase-example=java-springdata-quickstart-repo&source=github)](https://gitpod.io/#https://github.com/couchbase-examples/java-springdata-quickstart)
+## Table of Contents
+
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Source Code](#source-code)
+  - [Install Dependencies](#install-dependencies)
+  - [Database Server Configuration](#database-server-configuration)
+  - [Environment Variables](#environment-variables)
+- [Running the Application](#running-the-application)
+  - [Directly on the machine](#directly-on-the-machine)
+  - [Docker](#docker)
+- [What We'll Cover](#what-well-cover)
+  - [Document Structure](#document-structure)
+  - [Code Organization](#code-organization)
+- [Data Model](#data-model)
+- [Let's Review the Code](#lets-review-the-code)
+  - [Integration Tests](#integration-tests)
+  - [Repository](#repository)
+  - [Model](#model)
+  - [Controller](#controller)
+  - [Service](#service)
+- [Route Specifications](#route-specifications)
+  - [GET Route](#get-route)
+  - [POST Route](#post-route)
+  - [PUT Route](#put-route)
+  - [DELETE Route](#delete-route)
+- [Route Workflow](#route-workflow)
+  - [GET Route Workflow](#get-route-workflow)
+  - [POST Route Workflow](#post-route-workflow)
+  - [PUT Route Workflow](#put-route-workflow)
+  - [DELETE Route Workflow](#delete-route-workflow)
+- [Custom SQL++ Queries](#custom-sql-queries)
+- [Running The Tests](#running-the-tests)
+- [Project Setup Notes](#project-setup-notes)
+- [Contributing](#contributing)
+- [Appendix](#appendix)
+  - [Extending API by Adding New Entity](#extending-api-by-adding-new-entity)
+  - [Running Self Managed Couchbase Cluster](#running-self-managed-couchbase-cluster)
+  - [Swagger Documentation](#swagger-documentation)
+
+## Getting Started
+
+### Prerequisites
 
-## Overview
-This quickstart tutorial will review the basics of using Couchbase by building a simple Spring Data REST API that stores user profiles is used as an example.
-
-## What We'll Cover
-- [Cluster Connection Configuration](#cluster-connection-configuration) – Configuring Spring Data to connect to a Couchbase cluster.
-- [Database Initialization](#database-initialization) – Creating required database structures upon application startup
-- [CRUD operations](#create-or-update-a-profile) – Standard create, update and delete operations.
-- [Custom SQL++ queries](#search-profiles-by-text) – Using [SQl++](https://www.couchbase.com/sqlplusplus) with Spring Data.
-
-## Useful Links
-- [Spring Data Couchbase - Reference Documentation](https://docs.spring.io/spring-data/couchbase/docs/current/reference/html/)
-- [Spring Data Couchbase - JavaDoc](https://docs.spring.io/spring-data/couchbase/docs/current/api/)
-
-## Prerequisites
 To run this prebuild project, you will need:
+
```

Comment:
The focus should be on Capella not the local Couchbase server. It is just an option in the appendix. Check against other tutorials.

Reply:
adapted